index.template.html

```diff
@@ -382,6 +382,8 @@
                                     <option value="-6">-6</option>
                                 </select>
                             </td>
+                            <td><span class="totalMod">---</span>
```

Comment:
Are these dashes needed?

Reply:
the dashes are just a placeholder to show that something should be there while it's calculating, not strictly necessary but there were numbers manually written in for the other stats so I thought something should go there